community-membership.md

```diff
@@ -180,10 +180,9 @@ files.
 
 ### Requirements
 
-The process for becoming a Maintainer should be defined in the SIG charter of
-the SIG owning the subproject. Unlike the roles outlined above, the Owners of a
-subproject are typically limited to a relatively small group of decision makers
-and updated as fits the needs of the subproject.
+Unlike the roles outlined above, the Owners of a subproject are typically
```

Comment:
nit: are we calling these people Owners or Maintainers? (I prefer Maintainers since we've already used that term above) it seems (?) like we're using those terms interchangeably here.

Reply:
I took the wording from the "requirements" section above. I'll change them both.